en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -266,78 +265,81 @@ all_data <- all_data %>% mutate(
+### Interpreting Wordfish
 
-Secondly, Wordfish identifies which specific, unique categories of words the model used to perform this document scaling, and [arrays](LINK to definition) them along a similar dimension. The placement of words along this scale is more inherently informative that the scaling of documents, because you can see which words are driving the ‘formation’ of the scale by seeing which words appear at each end of it.
+A Wordfish model gives two kinds of information, without the need for any prior pre-coding of documents or 'supervision'.
 
-The scaling of documents is less inherently meaningful without additional information about those documents.  However, including metadata about the source of documents (in this case, the partisanship of the video contributing the comment) can be very helpful for determining if the greatest differences in types of documents exists within a given data source, or between data sources.  For example, if a corpus of comments from several videos is analyzed, it can be useful to see if comments from some videos are clustered together, and apart from comments on other videos.
+First, Wordfish gives information about how documents (in this case, individual comments) are best differentiated from each other, by scaling them along a single dimension. In essence, the model collects comments that are similar to each other on each end of the dimension, based on the kinds and frequencies of words used in those comments. Comments on far opposite ends of this scale in particular types of discursive contexts may be characterized by the inclusion of different sets of unique words, indicating focus on different kinds of concepts. 
 
-### Latent Meaning
-Although Wordfish was initially developed by political scientists for researching political ideology, there is nothing inherently political about the dimension revealed by Wordfish. Wordfish can be used to extract "latent" dimensionality (based on broad differences in words used) existing within any corpus. The substantive interpretation of this latent dimension depends entirely on broad trends in the typical contents of the documents comprising your research corpus.
+Secondly, Wordfish identifies which specific, unique categories of words the model used to perform this document scaling, and [arrays](LINK to definition) them along a similar dimension. The placement of words along this scale is more inherently informative that the scaling of documents, because you can see which words are driving the formation of the scale by seeing which words appear at each end of it.

```

Comment:
Could you also provide a suitable definition for 'arrays' here?

Reply:
Let's not use the word array, but lets rewrite this section as follows because it was not written well:

"Secondly, Wordfish develops a scale to describe the polarization of words and documents within a corpus, and then arranges those words along an analogous dimension to the document scale. The placement of words along this scale is more inherently informative than the scaling of documents..."